### PR TITLE
Update CONTRIBUTING.md with a proper Splunk CLA link

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

File: specification/templates/CONTRIBUTING.md [hunk 1]
```diff
@@ -75,6 +75,5 @@ any 'help wanted' issues is a great place to start.
 See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
 confirm the licensing of your contribution.
 
-We may ask you to sign a [Contributor License Agreement
-(CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) for larger
-changes.
+## CLA
+All contributors must execute the [Splunk Contributor License Agree (CLA)](https://www.splunk.com/en_us/form/contributions.html) form.
```
Comment (pellared) [inline specification/templates/CONTRIBUTING.md:79]:
```suggestion

All contributors must sign off the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html) form.
```

File: specification/templates/CONTRIBUTING.md [hunk 2]
```diff
@@ -75,6 +75,6 @@ any 'help wanted' issues is a great place to start.
 See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
 confirm the licensing of your contribution.
 
-We may ask you to sign a [Contributor License Agreement
-(CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) for larger
-changes.
+## CLA
+
+All contributors must sign the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html).
```
Comment (pellared) [inline specification/templates/CONTRIBUTING.md:80]: I just checked and the whole file is written in "2nd form".
Therefore it would be better to rephrase it to:

```suggestion
Before contributing, you must sign the [Splunk Contributor License Agreement (CLA)](https://www.splunk.com/en_us/form/contributions.html).
```

Also maybe it would be better just to put it as a bullet in the `Contributing via Pull Requests` section?

File: specification/templates/CONTRIBUTING.md [hunk 3]
```diff
@@ -75,6 +75,6 @@ any 'help wanted' issues is a great place to start.
 See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
 confirm the licensing of your contribution.
```
Comment (pellared) [inline specification/templates/CONTRIBUTING.md:76]: Could we remove `We will ask you to confirm the licensing of your contribution.` if one already has to sign the CLA?

**Review by pellared — APPROVED** [review]
For me this is good enough

**Comment by pellared** [comment]
@MrAlias PTAL

File: specification/templates/CONTRIBUTING.md [hunk 4]
```diff
@@ -75,6 +75,6 @@ any 'help wanted' issues is a great place to start.
...
 
-We may ask you to sign a [Contributor License Agreement
-(CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) for larger
-changes.
+## CLA
```
Comment (MrAlias) [inline specification/templates/CONTRIBUTING.md:78]: Is a new section needed? The contributor license agreement seems appropriate to be included in the `Licensing` section itself.

```suggestion
```

If not, can we make it a sub-section of `Licensing` and not use an acronym here?

```suggestion
### Contributor License Agreement
```

Comment (mateuszrzeszutek) [reply specification/templates/CONTRIBUTING.md]: Good point, made it a subsection.

**Review by MrAlias — APPROVED** [review]
